RealmSwift/Tests/ObjectTests.swift

```diff
@@ -42,7 +42,9 @@ class ObjectTests: TestCase {
 
         let queue = dispatch_queue_create("background", DISPATCH_QUEUE_SERIAL)
         dispatch_async(queue) {
-            XCTAssertNotEqual(Realm(), persisted.realm!)
+            autoreleasepool {
```

Comment:
I _believe_ it's because we are capturing `persisted.realm` in the block, but I'm not entirely sure